src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Controllers/AdminController.cs

```diff
@@ -114,6 +115,18 @@ public ActionResult EndTest(int testId)
             return this.RedirectToAction("Index");
         }
 
+        [HttpPost]
+        public ActionResult CalculateResults(int weekNumber)
+        {
+            if (weekNumber < 1 && weekNumber > 4)
+            {
+                return this.BadRequest("Błędny numer tygodnia.");
+            }
+
+            Process.Start(@"c:\\Windows\\System32\\cmd.exe", weekNumber.ToString(CultureInfo.CurrentCulture.DateTimeFormat));
```

Comment:
are you sure this path is correct for the server?
I suggest adding it to configuration ex. in appsettings file

Reply:
Path is incorrect - Patryk agreed to rewrite the code.

Reply:
so we can add a comment here to remember to fix it :)